lib/myfinance/entities/payable_account_collection.rb

```diff
@@ -0,0 +1,14 @@
+module Myfinance
+  module Entities
+    #
+    # A wrapper to Myfinance entities collection
+    #
+    class PayableAccountCollection < Collection
+      def build_collection
+        response.parsed_body.each do |attributes|
+          collection.push(Myfinance::Entities::PayableAccount.new(attributes['payable_account']))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.